consensus/state.go

```diff
@@ -71,23 +92,31 @@ func CreateGenesisState(genesisTime Timestamp) (s *State) {
 	// Fill out the consensus informaiton for the genesis block.
 	s.currentBlockID = genesisBlock.ID()
 	s.currentPath[BlockHeight(0)] = genesisBlock.ID()
-	s.unspentOutputs[genesisBlock.MinerPayoutID(0)] = SiacoinOutput{
-		Value:     CalculateCoinbase(0),
-		SpendHash: ZeroAddress,
+	s.siacoinOutputs[genesisBlock.MinerPayoutID(0)] = SiacoinOutput{
+		Value:      CalculateCoinbase(0),
+		UnlockHash: ZeroAddress, // TODO: change to Nebulous Genesis Siacoin SpendHash Address
 	}
-	s.unspentSiafundOutputs[OutputID{0}] = SiafundOutput{
-		Value:            NewCurrency64(SiafundCount),
-		SpendHash:        ZeroAddress, // TODO: change to Nebulous Genesis Address
-		ClaimDestination: ZeroAddress, // TODO: change to Nebulous Genesis Address
+	s.siafundOutputs[SiafundOutputID{0}] = SiafundOutput{
+		Value:           NewCurrency64(SiafundCount),
+		UnlockHash:      ZeroAddress, // TODO: change to Nebulous Genesis Siafund SpendHash Address
+		ClaimUnlockHash: ZeroAddress, // TODO: change to Nebulous Genesis ClaimDestination Address
 	}
 
 	return
 }
 
+// RLock will readlock the state.
+//
+// TODO: Add a safety timer which will auto-unlock if the readlock is held for
+// more than a second. (panic in debug mode)
```

Comment:
hmm. Doesn't seem like a perfect solution, but I suppose the debug output will be helpful.